docs/3-customizing-own-project.md

```diff
@@ -37,6 +37,90 @@ If you have built the sample project and have the pipeline working, it is very s
 
 1. Check in your changes and raise a PR to merge them into the master branch.
 
+## Enabling the F-measure testing capability
+
+The **LUIS F-measure testing** job measures performance of your LUIS model. This job performs the equivalent of the [LUIS batch testing](https://docs.microsoft.com/azure/cognitive-services/LUIS/luis-how-to-batch-test) that you can perform in the LUIS portal. It uses the [NLU.DevOps](https://github.com/microsoft/NLU.DevOps/blob/master/docs/Analyze.md) tool to execute the tests defined in the `luis-app/tests/verificationtests.json` test file against the newly built model and calculates the test results file, `statistics.json`, from which can be calculated the F1 score (or [F-measure](https://en.wikipedia.org/wiki/F-measure)). The results file from every run is saved to Azure blob storage and it contains:
```

Comment:
Super clear intro with exactly the right amount of info vs. external links

Reply:
Nit. "from which can be calculated the F1 score (or F-measure)" -> "from which the F1 score (or F-measure) can be calculated"

Reply:
You say potatoe, I say potartoe :)  NP - updated.